app/src/main/java/live/ditto/dittotoolsapp/HeartbeatView.kt

```diff
@@ -22,14 +22,15 @@ import live.ditto.dittoheartbeat.startHeartbeat
 import java.util.*
 
 
+@RequiresApi(Build.VERSION_CODES.O)
 @Composable
 fun ShowHeartbeatData(ditto: Ditto) {
 
     var heartbeatInfo by remember { mutableStateOf<DittoHeartbeatInfo?>(null) }
     var healthMetricProviders: MutableList<HealthMetricProvider> = mutableListOf()
     val diskUsageViewModel = DiskUsageViewModel()
     diskUsageViewModel.isHealthyMBSizeLimit = 2048 //2GB worth of data
-    healthMetricProviders.add(diskUsageViewModel)
+//    healthMetricProviders.add(diskUsageViewModel)
```

Comment:
This is commented out for now due to https://github.com/getditto/DittoAndroidTools/issues/93